cypress/plugins/index.js

```diff
@@ -1,9 +1,24 @@
 module.exports = (on, config) => {
-  on('before:browser:launch', (browser = {}, launchOptions) => {
-    if (browser.family === 'chromium' && browser.name !== 'electron') {
-      launchOptions.args.push('--disable-dev-shm-usage')
+  // on('before:browser:launch', (browser = {}, launchOptions) => {
+  //   if (browser.family === 'chromium' && browser.name !== 'electron') {
+  //     launchOptions.args.push('--disable-dev-shm-usage')
```

Comment:
why dont we need this anymore? i remember it getting rid of a lot of weird flakiness (where the page didn't even render before running a test iirc)

Reply:
I have no idea- this is your tests, feel free to improve after this PR.

Reply:
lol do your changes still work when you dont delete the code that is in there to keep it running?

Reply:
oops meant to switch this back when I was done, thanks